content/resources/chapter-5/verify-signature-5.tsx

```diff
@@ -68,15 +57,16 @@ function verify(sig_r, sig_s, pubkey_x, pubkey_y, msg) {
 
     return x1;
   }
-  // Implement ECDSA!
-  //   u1 = m / s mod n
-  //   u2 = r / s mod n
-  //   R = G * u1 + A * u2
-  //   r == x(R) mod n
   const sig_s_inverted = invert(sig_s);
+  // Implement ECDSA!
+  //   u1 = msg * s mod n
```

Comment:
```suggestion
  //   u1 = msg * s^(-1) mod n
```

This should be msg * the **inverse** of s (recall that 1/n is the same as 1 * n^-1, I haven't thought of this in at least 10 years and was caught off guard when I originally did this chapter :laughing:). 

I personally would restore the original `// u1 = m / s mod n` code comment because I think that's how it appears in the cryptography documentation. I see that you kept it this way for the python solution. Not sure if @pinheadmz wants to give a second opinion.